app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -279,7 +279,7 @@
         vm.selectedCharacter = 0;
         vm.lockeditems = { Helmet: null, Gauntlets: null, Chest: null, Leg: null, ClassItem: null, Artifact: null, Ghost: null };
         vm.lockedperks = { Helmet: {}, Gauntlets: {}, Chest: {}, Leg: {}, ClassItem: {}, Artifact: {}, Ghost: {} };
-        vm.excludeditems = [];
+        vm.excludeditems = _.filter(vm.excludeditems, function(item) { return item.id === '6917529101951964046'; });
```

Comment:
we'll want to do this by hash `2672107540` as `id` is unique to each player and `hash` is the unique item identifier.